### DISCUSSION TITLE
Rename the GNU toolchain wrappers

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#976] [Ideas]
Use the `og` prefix. For example: `gccocc` will become `ogcc`.

**Comment by GitMensch** [comment]
Why?

**Reply by ghost** [reply]
A more systematic way to name the wrappers. Easier to remember and use. More efficient than `the original tool name` + `occ` suffix.

**Comment by GitMensch** [comment]
I find the current way easier to remember and use...
As you don't type .exe I suggest you too create minimal wrapper scripts with your preferred names as .cmd and just have that as

@echo off
Toolnameocc %*

**Reply by ghost** [reply]
Current name is too long and sometimes confusing. You have been with the project for a long time so you are familiar with it. The goal is people completely new to this project could tell what the binary is doing by just looking at the name, without having to consult any documentation.

**Reply by ghost** [reply]
For example, `omslink` is better than `linkocc` in my opinion because it better resembles the MSVC `link`. Prefix is always easier to spot than suffix. It will not take it long for a completely new user to guest what the binary is doing.
